test/reconstruction/gapfill_minimum_reactions.jl

```diff
@@ -6,20 +6,24 @@
 
     (m1, m2, m3, m4, m5, m6, m7, m8) = [Metabolite(id = "m$i") for i = 1:8]
 
-    @add_reactions! model begin
-        "r1", nothing → m1, 0, 1
-        "r2", m1 ↔ m2, -10, 100
-        "r3", m1 → m3, 0, 100
-        "r4", m2 ↔ m4, 0, 100
-        # "r5", m3 → m4, 0, 100
-        "r6", m4 → nothing, 0, 100
-        # "r7", m2 → m7 + m6, 0, 100
-        "r8", m7 → m8, 0, 100
-        "r9", m8 → nothing, 0, 100
-        # "r10", m6 → nothing, 0, 100
-        "r11", m2 + m3 + m7 → nothing, 0, 100
-        "r12", m3 → m5, -10, 10
-    end
+    add_reactions!(
+        model,
+        [
+            # Reaction("r1", Dict("m1" => 1), :forward; upper_bound = 1),
```

Comment:
rm comment

Reply:
I kept the commented ones to make it obvious what is actually being tested (the same reactions later get added back. I think this makes it clear what is expected. I put a comment in the code for future devs

Reply:
ok cool